utils/datautils.js

```diff
@@ -48,6 +48,18 @@ function getClaimById(claimId) {
   return db.Claim.findById(claimId)
 }
 
+function getConflictingClaimByPr(pullRequestId) {
```

Comment:
Not the correct logic. just checking the presence of id does not decide the id is for pr or issue.

Reply:
I think i named the parameter wrongly in this one.
for a the url
`https://github.com/coding-blocks/boss/pull/279`
this is the value of the parameter
`coding-blocks/boss/pull/279`
and not only `279`

Reply:
![image](https://user-images.githubusercontent.com/14032427/82268853-98ba3500-998d-11ea-9ef9-73803caa5bc3.png)

Reply:
This checks the unique part of the URL, which is of the type `user/repository/pull/id`. The above confusion arose due to improper variable naming, which I've fixed now.